scopes.md

```diff
@@ -0,0 +1,147 @@
+
+Also it is possible that scopes depend on other scopes. It means that requesting and granting of a specific scope is only possible along with the scope it depends on. For example the `private:read` and `private:modify` scope which grants reading private posts permission depends on `contacts:read` scope.
+
+Each API endpoint requires at least one scope to be granted. If the access token used for making a request wasn't granted the scope of the respective endpoint then the response will have a `403` status code and no actual content.
+
+In this specification each endpoint description contains the name of the scope required to access this endpoint. Most of the time this is only one scope. However for some types of data like posts the required scope depends on the data visibility (public vs private). Besides this, for some endpoints, like streams the dataset in response depends on the set of granted scopes.
+
+## Scopes
+
+### contacts:read
+
+This scope gives access to the following requests:
+- Read aspect list
+- Read aspects membership
+- Read private user profiles of contacts
+- Read contacts of contacts when allowed by contact's aspect setting
+- User search will include contacts who are hidden from public search
+
+### contacts:modify
+
+This scope gives access to the following requests:
+- Add new aspects
+- Rename aspects and change aspects properties
+- Delete aspects
+- Add/remove person to/from aspects
+
+### conversations
+
+This scope gives access to private messaging, i.e. the following requests:
+- Create conversations
+- Send private messages
+
+### email
+
+This scope is defined by the OpenID Connect spec. It allows reading user's own email address using the UserInfo route or from the ID Token.
+
+### interactions
+
+This scope gives access to the following requests:
+- Create/delete comments
+- Report posts/comments
+- Post/remove likes
+- Subscribe/mute posts
+- Hide posts from streams
+- Voting in polls
+
+### notifications
+
+This scope gives access to reading notifications.
+
+### openid
+
+This scope is defined by the OpenID Connect spec. This is a mandatory scope. It allows using OpenID Connect authentication.
+
+### private:read
+
+This scope gives access to the following requests:
+- Read private posts
+- Read private posts interactions (likes, comments)
+- Reading streams: main stream, aspects stream, mention stream, activity stream, liked stream, commented stream
+
+Note: the `private:read` scope depends on `contacts:read` and can only be granted when `contacts:read` is granted too.
+
+### private:modify
+
+This scope gives access to the following requests:
+- Create private posts
+- Post photos privately
+- Delete private posts
+- Delete private photos
+
```

Comment:
Should we have "Delete private posts interactions (likes, comments)"?

Reply:
I think that sounds good.

Reply:
I think we're good here :)